src/renderer/components/MainPage.tsx

```diff
@@ -529,11 +537,16 @@ class MainPage extends React.PureComponent<Props, State> {
                     <button
                         className='three-dot-menu'
                         onClick={this.openMenu}
+                        onMouseOver={this.focusThreeDotsButton}
```

Comment:
Does this properly handle when focusing an element with the keyboard? Perhaps we could make it so that instead of adding a separate state to track focusing, we instead just blur the menu button on `APP_MENU_WILL_CLOSE`

Reply:
0/5 on that as long as it still works with the keyboard. I wouldn't be surprised if there was some weird issue with blurring it where maybe it works differently in a BrowserView than I'm used to in an actual browser, but I don't have enough Electron experience to know for sure